roxie/ccd/ccdstate.cpp

```diff
@@ -2372,6 +2372,12 @@ class CRoxiePackageSetManager : implements IRoxieQueryPackageManagerSet, impleme
                 topology->setPropInt("@leafCacheMem", leafCacheMB);
                 setLeafCacheMem(leafCacheMB * 0x100000);
             }
+            else if (stricmp(queryName, "control:legacyNodeCache")==0)
+            {
+                bool legacyNodeCache = control->getPropBool("@val", true);
+                topology->setPropInt("@legacyNodeCache", legacyNodeCache);
+                setLegacyNodeCache(legacyNodeCache);
```

Comment:
Would this require existing node/leaf/blob cache flush/re-init ?

Reply:
no, I think it can change dynmically.